packages/jsii-python-runtime/tests/test_compliance.py

```diff
@@ -49,8 +49,7 @@
 #       Tests as closely as possible to make keeping them in sync easier.
 
 # These map distinct reasons for failures, so we an easily find them.
-xfail_async = pytest.mark.xfail(reason="Implement async methods", strict=True)
-xfail_callbacks = pytest.mark.xfail(reason="Implement callback support", strict=True)
+xfail_callbacks = pytest.mark.skip(reason="Implement callback support")
```

Comment:
I had to switch this to a skip, because running these tests and failing on them left the process in a bad state.